packages/cloud-config/oem/02_upgrades.yaml

```diff
@@ -35,17 +33,15 @@ stages:
                     verify: false
                     urls:
                     - "quay.io/costoolkit/releases-:FLAVOR::ARCH:"
-         permissions: 0600
-         owner: 0
-         group: 0
      - name: "Setup upgrade channel"
        files:
-       - path: /etc/cos-upgrade-image
-         permissions: 0600
+       - path: /etc/elemental/config.yaml
```

Comment:
This needs adaptation in docs (that we don't support /etc/cos-upgrade-image , and that now it's /etc/elemental/config.yaml )